src/sections/qualitat.html

```diff
@@ -1,18 +1,17 @@
-
-    <div class=" relative z-0 px-4 -top-44 pt-52 lg:pt-52 pb-48 md:px-10 xl:px-36 ojemba-oblique2 2xl:grid justify-center">
-        <div> 
-            <p class="text-white md:w-1/2">
-                <span class="font-railwayBold text-xl leading-3 ">Qualität steht bei uns an erster Stelle. Mit unserem
-                    <span class="text-green">Trainingsprogramm</span> direkt Vorort werden die  juniorigen Mitarbeitenden auf das projektgeschäft <br> vorbereitet.</span> 
-                <br> <br>
-                <span class="font-extralight text-base leading-none">
-                    kurze Einführung und den Bezug zu MW darstellen At vero eos et accusam et justo duo
-                    dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est L vero eos et
-                    accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata
-                    sanctus est L
-                </span>
-            </p>
-        </div>
+<div class="relative z-0 px-4 -top-44 pt-52 lg:pt-52 pb-48 md:px-10 xl:px-36 ojemba-oblique2 2xl:grid justify-center">
+    <div> 
+        <p class="text-white md:w-1/2">
+            <span class="font-railwayBold text-xl leading-3 ">Qualität steht bei uns an erster Stelle. Mit unserem
```

Comment:
You can remove that space at the end of this class, I think there is no need for it.